### ISSUE TITLE
Water World Mode - Incorrect Block State

### ISSUE THREAD

**Issue by MadDog443** [chunky-dev/chunky#1247]
On Water World Mode there's been this issue that has persisted for a long time which honestly, I think has a decent effect on renders in terms of it standing out. On the Edge of chunks water has the wrong block state so if you use Water World Mode, it gives you this really annoying line into the abyss.
![image](https://user-images.githubusercontent.com/52639083/165677426-f645e6c2-dbf5-4b6e-8426-9cd9fa9b85fb.png)

**Comment by MadDog443** [comment]
The only other issue I could see happening is not being able to set a **sea floor height** or **block palette w/ noise map** (kind of like procedural water) to avoid the water world looking off in the render due to it reflecting light differently. _This second comment is mainly a feature suggestion to improve the Water World Feature so feel free to ignore it._
As seen here:
![image](https://user-images.githubusercontent.com/52639083/165735329-af6dbdd6-867a-496a-ba66-71de35b50491.png)

**Comment by Peregrine05** [comment]
A workaround at the moment is to disable `Hide water plane in loaded chunks`. That extends the water plane over the incorrect water model and hides the gap.

**Comment by MadDog443** [comment]
> A workaround at the moment is to disable `Hide water plane in loaded chunks`. That extends the water plane over the incorrect water model and hides the gap.

Yeah, that would not work for me very well.... too many things visible below water level.

**Comment by MadDog443** [comment]
A very belated cheers to this fix!
